core/src/main/resources/xml/xsl/adapter2mermaid.xsl

```diff
@@ -0,0 +1,1073 @@
+			<xsl:with-param name="isValidationFlow" select="if($isValidationFlow castable as xs:boolean) then ( $isValidationFlow ) else ( count(isValidationFlow) = 1 )"/>
+			<xsl:with-param name="text" select="@name"/>
+			<xsl:with-param name="destination" select="@path"/>
+			<xsl:with-param name="specialForward" select="if(string-length(@specialForward) != 0) then (@specialForward) else ('')"/>
+			<xsl:with-param name="specialForwardValue" select="if(specialForwardValue) then (specialForwardValue) else (@specialForwardValue)"/>
+<!--			<xsl:with-param name="goesToOtherAdapter" select="if(string-length(@goesToOtherAdapter) != 0) then (@goesToOtherAdapter) else ('')"/>-->
+<!--			<xsl:with-param name="goesToOtherAdapter" select="@goesToOtherAdapter"/>-->
+		</xsl:call-template>
+	</xsl:template>
+
+	<xsl:template name="forward">
+		<xsl:param name="pipeline"/>
+		<xsl:param name="id"/>
+		<xsl:param name="specialIds"/>
+		<xsl:param name="flowHistory"/>
+		<xsl:param name="origin"/>
+		<xsl:param name="text"/>
+		<xsl:param name="destination"/>
+		<xsl:param name="isValidationFlow"/>
+		<xsl:param name="specialForward" select="''"/><!--Used to specify special forwards such as input and output wrappers and validators.-->
+		<xsl:param name="specialForwardValue"></xsl:param><!--Used to supply nessecary information when this forward is a special one.-->
+<!--		<forwardTest>-->
+<!--			<id><xsl:value-of select="$id"/></id>-->
+<!--			<specialIds><xsl:value-of select="$specialIds"/></specialIds>-->
+<!--			<flowHistory><xsl:copy-of select="$flowHistory"/></flowHistory>-->
+<!--			<origin><xsl:value-of select="$origin"/></origin>-->
+<!--			<text><xsl:value-of select="$text" disable-output-escaping="yes"/></text>-->
+<!--			<destination><xsl:value-of select="$destination"/></destination>-->
+<!--			<isValidationFlow><xsl:value-of select="$isValidationFlow"/></isValidationFlow>-->
+<!--			<specialForward><xsl:copy-of select="$specialForward"/></specialForward>-->
+<!--			<specialForward><xsl:value-of select="$specialForward"/></specialForward>-->
+<!--			<specialForwardValue><xsl:copy-of select="$specialForwardValue"/></specialForwardValue>-->
+<!--			<specialForwardValue><xsl:value-of select="$specialForwardValue"/></specialForwardValue>-->
+<!--			<testSpecialForward><xsl:value-of select="$specialForward != 'differentAdapter'"/></testSpecialForward>-->
+<!--			<testSpecialForward><xsl:value-of select="$specialForward = 'differentAdapter'"/></testSpecialForward>-->
+<!--			<testSpecialForward><xsl:value-of select="string-length($specialForward)"/></testSpecialForward>-->
+<!--		</forwardTest>-->
+
+<!--		<forwardTest2><xsl:copy-of select="ancestor::pipeline"/></forwardTest2>-->
+<!--		<xsl:text>&#10;</xsl:text>-->
+		<element isValidationFlow="{$isValidationFlow}">
+			<identification>
+				<!--If this forward goes to a different adapter, use the id supplied in $specialForwardValue, else use this adapters id.
```

Comment:
Hoe kan een forward naar een andere adapter gaan? Of gaat het hier niet om onze forward maar een 'Mermaid-forward'?